examples/orderbook.py

```diff
@@ -1,10 +1,8 @@
 import json
-from collections import OrderedDict
 from dataclasses import dataclass, field
 from datetime import timedelta
-from typing import List, Optional
+from typing import Dict, List, Optional
 
-# pip install websockets
```

Comment:
This comment should problably stay.

To explain why it's here, we don't have an explicit `requirements.txt` file for examples, so we write the exact dependencies as a comment. In this case it's trivial, python module and package on pypi have the same name, but sometimes they don't, and we used to have errors every time we installed the wrong package with a similar name, so we add a comment for any external package. We could remove this, but we should have a `requirements.txt` to keep track of which packages we use in all the examples.
*edit*: I see that some other examples don't have all the packages specified, but I'd rather add it there than remove it here